Enhanced the use of namespaces (#195)
EMMC-ASBL/tripper: docs/tutorial.md

Both of these features requires loading an ontology.
The following example shows how to create an EMMO namespace with IRI checking.
The keyword argument `label_annotations=True` enables access by `skos:prefLabel`, `rdfs:label` or `skos:altLabel`.
What labels to use can also be specified explicitly.
The `check=True` enables checking for existing IRIs.

```python
>>> EMMO = ts.bind(
...     "emmo", "https://w3id.org/emmo#",
...     label_annotations=True,
...     check=True,

> ... )

Reviewer: This is a nice feature that is good to document in this simple example.
Don't remove it completely

Author: It is not removed. I only removed the `triplestore_url` argument, since it is not needed any more. All the labels are now fetch from the IRI "https://w3id.org/emmo#" provided as the second argument. 

Enabling simplifications like this, is part of the goal of this PR :-)

Reviewer: It is removed in the tutorial. I think it should stay, either in the same example or add another just after to point out this very important possiblity.

Author: Added a reformulation of the comment about the use of the `triplestore` argument below the code example.

```

The `query()` and `update()` methods can be used to query and update the triplestore using SPARQL.
See the next section.

Writing SPARQL queries using Tripper
------------------------------------
A challenge with ontologies using numerical IRIs is that SPARQL queries become difficult to read and understand.
This challenge is greatly mitigated by using the `label_annotations` feature of Tripper namespaces.
The example below shows how to write and execute a SPARQL query with Tripper that finds the IRI and unit symbol of all length units.

> Note how:

Reviewer: ```suggestion
Note:
```

1. EMMO classes and properties are written as `{EMMO.LengthUnit}`, which would expand to `https://w3id.org/emmo#EMMO_b3600e73_3e05_479d_9714_c041c3acf5cc`.

> 2. That the curly brackets after the `WHERE` clause has to be written `{{`, `}}` because the query is an f-string.

Reviewer: ```suggestion
2. The curly brackets after the `WHERE` clause have to be written `{{`, `}}` because the query is an f-string.
```
